__tests__/integration.test.js

```diff
@@ -23,6 +22,24 @@ describe('GET: api/topics', () => {
     });
 });
 
+describe('GET: /api', () => {
+    test('200: responds successfully with a list of all endpoints', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({ body }) => {
+            const endpoints = require('../endpoints.json')
```

Comment:
You're halfway there with this logic! This can be moved to the top of the file, then rather than having to do the logic underneath in this test, we can just expect the response that comes back to equal the contents of the json file 👍